gitlabform/lists/projects.py

```diff
@@ -112,6 +99,36 @@ def _get_projects(self, target: str, groups: Groups) -> Projects:
                 OmissionReason.ARCHIVED,
                 archived_projects_from_configuration_not_from_groups,
             )
+        else:
+            # in all other cases, we also need to look for projects in the config
+            # that are being transferred to a different namespace
+            for project in projects_from_configuration_not_from_groups:
+                # if the target is a group, and the project is not in the group
+                if (
+                    len(groups.get_effective()) == 1
+                    and target != project.rsplit("/", 1)[0]
+                ):
+                    debug(
+                        "Ignore project '%s', since it's not in target group '%s",
```

Comment:
Should we remove the term `group`? This will be applicable when we do `gitlabform ALL`, right? Not sure if mention of `group` in that scenario gets confusing. Thoughts?

Reply:
This is specifically for group targets, updated the if condition to be more explicit.

Reply:
Thanks for clarifying.